doc/in-progress/sui-kiosk.md

```diff
@@ -0,0 +1,126 @@
+---
+title: Sui Kiosk
+---
+
+Sui Kiosk is a primitive, or building block (a module in the Sui Framework), you can use to build a trading platform for digital assets. Sui Kiosk supports adding digital assets that you can store and list for sale to other users. You can also define rules for the kiosk as part of a transfer policy that controls how purchasers can use the asset after purchase. 
+To add digital assets to your kiosk to list for sale, you must first create and publish a package to Sui as part of a programmable transaction block. The package must define a type (T) and include a transfer policy (a `TransferPolicy` object), that determines whether, and how, users can interact with the assets in your kiosk. You can specify different requirements for each asset and transaction. To learn more about transfer policies, see [Transfer Policy](link-to-topic).
```

Comment:
> The package must define a type (T) and include a transfer policy

Package does not need to include the TransferPolicy; it can (and better) be created later using the `Publisher` object (I think we should link to it here).